MMA Removal Utility/SingleTenantSetupInstallation.md

```diff
@@ -0,0 +1,302 @@
+- Creates or updates the resources with MIs assigned.
+- Creates or updates the monitoring dashboard.
+- Configures target scopes.
+
+Before executing the installation command, please log in to Azure Account using the following PowerShell command.
+
+``` PowerShell
+
+# Connect to AzAccount
+# Note: Tenant Id *must* be specified when connecting to AzAccount
+
+$TenantId = "<TenantId>"
+Connect-AzAccount -Tenant $TenantId
+```
+
+``` PowerShell
+# -----------------------------------------------------------------#
+# Consolidated installation
+# -----------------------------------------------------------------#
+
+$SetupInstallation = Install-AzTSMMARemovalUtilitySolution ` 
+         -RemediationIdentityHostSubId <MIHostingSubId> `
+         -RemediationIdentityHostRGName <MIHostingRGName> `
+         -RemediationIdentityName <MIName> `
+         -TargetSubscriptionIds @("<SubId1>","<SubId2>","<SubId3>") `
+         -TargetManagementGroupNames @("<MGName1>","<MGName2>","<MGName3>") `
+         -TenantScope `
+         -SubscriptionId <HostingSubId> `
+         -ResourceGroupName <HostingRGName> `
+         -Location <Location> `
+         -AzureEnvironmentName <AzureEnvironmentName> `
+         -DisableUsageTelemetry
+```
+
+**Parameter details:**
+|Param Name|Description|Required?
+|----|----|----|
+|RemediationIdentityHostSubId| Subscription id in which remediation MI needs to be created.| Yes|
+|RemediationIdentityHostRGName| Resource group name in which remediation MI needs to be created. Default value is 'AzTS-MMARemovalUtility-RG'| No|
+|RemediationIdentityName| Name of the remediation MI.| Yes|
+|TargetSubscriptionIds| List of target subscription id(s) on which AzTS MMA Removal Utility Setup will run.| No|
+|TargetManagementGroupNames| List of target management group name(s) on which AzTS MMA Removal Utility Setup will run.| No|
+|TenantScope| Switch for tenant scope in case you want to scope the run at tenant level, configures scope and assigns appropriate roles at the tenant level using tenant id from the context of user.| No|
+|SubscriptionId| Subscription id in which setup is installed.| Yes|
+|ResourceGroupName| Resource group name in which remediation MI needs to be created. Default value is 'AzTS-MMARemovalUtility-Host-RG'| No|
+|Location| Location in which remediation MI needs to be created. For better performance, we recommend hosting the MI and AzTS MMA Removal Utility setup in the same location. Default value is 'EastUS2'| No|
+|AzureEnvironmentName| Azure environment in which MMA Removal Utility Solution needs to be installed. The acceptable values for this parameter are: AzureCloud, AzureGovernmentCloud. Default value is 'AzureCloud'| No|
+|DisableUsageTelemetry| DisableUsageTelemetry switch will not prompt for user preferences that captures usage data (stored in Microsoft servers). Avoiding this switch will help in improving the product quality and prioritize meaningfully on the highly used features.| No|
```

Comment:
Enabling this switch will help in improving the product quality and prioritize meaningfully on the highly used features.

Reply:
Removed usage telemetry.